tests/ast/data/col_alias.test

```diff
@@ -0,0 +1,20 @@
+## TEST CASE
+from snowflake.snowpark.functions import col
```

Comment:
I would suggest adding `import snowflake.snowpark.functions` in the test driver and using `functions.col` in your tests.

Reply:
Or because `col` is so special (i.e. common), adding that import as well (keeping .functions for other functions in the future).